packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+# 50.14.0 - Month DD, YYYY
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Remove channelService from core extension ([PR #2853](https://github.com/forcedotcom/salesforcedx-vscode/pull/2853))
+
+#### salesforcedx-vscode-soql
+
+- Do not emit message from custom editor to webview if soql has not changed ([PR #2818](https://github.com/forcedotcom/salesforcedx-vscode/pull/2818))
+
+## Fixed
+
+#### salesforcedx-utils-vscode
+
+- WorkspaceContext initialization across apex & replay-debugger packages ([PR #2868](https://github.com/forcedotcom/salesforcedx-vscode/pull/2868))
+
+#### salesforcedx-vscode-apex
+
+- WorkspaceContext initialization across apex & replay-debugger packages ([PR #2868](https://github.com/forcedotcom/salesforcedx-vscode/pull/2868))
+
+#### salesforcedx-vscode-apex-replay-debugger
+
+- WorkspaceContext initialization across apex & replay-debugger packages ([PR #2868](https://github.com/forcedotcom/salesforcedx-vscode/pull/2868))
+
+#### salesforcedx-vscode-soql
+
+- Do not show toggle command in palette ([PR #2867](https://github.com/forcedotcom/salesforcedx-vscode/pull/2867))
+
+- Disable remote query checks. Make it an experimental option ([PR #2847](https://github.com/forcedotcom/salesforcedx-vscode/pull/2847))
+
+- Revert auto-publish of SOQL extension  (#2859) ([PR #2843](https://github.com/forcedotcom/salesforcedx-vscode/pull/2843))
+
+- Fix local test environment ([PR #2857](https://github.com/forcedotcom/salesforcedx-vscode/pull/2857))
```

Comment:
You can remove soql changes since it isn't being published with the rest of the extensions

Reply:
Any objections to me adding changes made under salesforcedx-vscode-soql to the ignore list for the changelog?

Reply:
@sfsholden - I think that will really help. I hope it is easy to revert is once salesforcedx-vscode-soql goes GA.